src/storyscreen/part.cpp

```diff
@@ -240,7 +240,7 @@ bool part::resolve_wml_helper(const std::string& key, const vconfig& node)
 
 	// [background_layer]
 	if(key == "background_layer") {
-		background_layers_.push_back(node.get_parsed_config());
+		background_layers_.push_back(background_layer(node.get_parsed_config()));
```

Comment:
Instead of an explicit constructor call, could you change this to use `emplace` instead of `push_back`?